veggieseasons/lib/screens/list.dart

```diff
@@ -65,12 +69,14 @@ class ListScreen extends StatelessWidget {
           ),
         );
 
-        rows.addAll(_generateVeggieRows(appState.unavailableVeggies, prefs));
+        rows.addAll(_generateVeggieRows(appState.unavailableVeggies, prefs, inSeason: false));
 
         return DecoratedBox(
           decoration: BoxDecoration(color: Color(0xffffffff)),
-          child: ListView(
-            children: rows,
+          child: CupertinoScrollbar(
```

Comment:
Any reason this isn't in the Cupertino Widget Gallery? I didn't know it existed.

https://flutter.dev/docs/development/ui/widgets/cupertino

Reply:
Oops, I didn't mean to keep this in. We'd need to fix https://github.com/flutter/flutter/issues/13253 first. I'll add it to the catalog tomorrow though.